extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/KyuubiHiveConnectorConf.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.spark.connector.hive
+
+import java.util.Locale
+
+object KyuubiHiveConnectorConf {
+
+  import org.apache.spark.sql.internal.SQLConf.buildStaticConf
+
+  val EXTERNAL_CATALOG_SHARE_POLICY =
+    buildStaticConf("spark.sql.kyuubi.hive.connector.externalCatalog.share.policy")
+      .internal()
+      .doc(s"Indicates the share policy for the externalCatalog in the Kyuubi Connector, we use" +
+        "'all' by default. " +
+        "<li>ALL: Indicate to an external catalog is shared globally with the HiveCatalogs " +
+        "with the same catalogName. </li> " +
+        "<li>ONE: Indicate to an external catalog is used by only one HiveCatalog. </li> ")
```

Comment:
can we preserve the whole name `ONE_FOR_ONE` and `ONE_FOR_ALL` ?  it can help to recall the behavior